vars/karateTestUtils.groovy

```diff
@@ -181,17 +181,14 @@ void createFailedFeatureJiraIssue(KarateModuleExecutionSummary moduleSummary, Ka
         Labels     : [KarateConstants.ISSUE_LABEL]
     ]
 
-    def teamName = "Kitfox"
-    fields["Development Team"] = teamName
-
-//    def teamName = "TEAM_MISSING"
-//    def team = teamByModule[moduleSummary.name]
-//    if (team) {
-//        teamName = team.name
-//        fields["Development Team"] = teamName
-//    } else {
-//        echo "Module ${moduleSummary.name} is not assigned to any team."
-//    }
+    def teamName = "TEAM_MISSING"
+    def team = teamByModule[moduleSummary.name]
+    if (team) {
+        teamName = team.name
+        fields["Development Team"] = teamName
```

Comment:
Would recommend to avoid usage of space(s)...

Reply:
It's a legacy code that automatically populates Jira API message fields with the same name. It should be refactored but in the next tasks. Stay tuned.

Reply:
For this task, it was just uncommented.